### PR TITLE
configs: fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,*.pot,*.ts,./.git/logs,./share,./docs/man/es,./configs/attic,*_fr.*,*_es.*,README_es -L ans,ba,bulle,componentes,doubleclick,dout,dum,fo,halp,ihs,inout,parm,parms,ro,ser,te,ue,wille,wonte`

### CODE REVIEW

File: configs/sim/qtvcp_screens/woodpecker/woodpecker_postgui.hal [hunk 1]
```diff
@@ -57,7 +57,7 @@ net jogSelecty        axis.y.jog-enable      woodpecker.jog.wheel.y
 net jogSelectz        axis.z.jog-enable      woodpecker.jog.wheel.z
 net jogSelecta        axis.a.jog-enable     woodpecker.jog.wheel.a
 
-net jogIncrement      woodpecker.jog.wheel.incement
+net jogIncrement      woodpecker.jog.wheel.increment
```
Comment (luzpaz) [inline configs/sim/qtvcp_screens/woodpecker/woodpecker_postgui.hal:60]: source change

File: configs/sim/qtvcp_screens/woodpecker/woodpecker_postgui_ya.hal [hunk 2]
```diff
@@ -47,6 +47,6 @@ net jogSelecty        axis.y.jog-enable      woodpecker.jog.wheel.y
 net jogSelectz        axis.z.jog-enable      woodpecker.jog.wheel.z
 net jogSelecta        axis.a.jog-enable     woodpecker.jog.wheel.a
 
-net jogIncrement      woodpecker.jog.wheel.incement
+net jogIncrement      woodpecker.jog.wheel.increment
```
Comment (luzpaz) [inline configs/sim/qtvcp_screens/woodpecker/woodpecker_postgui_ya.hal:50]: source change

**Review by luzpaz — COMMENTED** [review]
Please review the source code changes closely

**Comment by c-morley** [comment]
Thanks

**Pull request merged**